Source/Core/FeatureDetection.js

```diff
@@ -163,6 +163,14 @@ define([
         return isFirefox() && firefoxVersionResult;
     }
 
+    var isNodeJsResult;
+    function isNodeJs() {
+        if (!defined(isNodeJsResult)) {
+            isNodeJsResult = typeof process === 'object' && Object.prototype.toString.call(process) === '[object process]'; // eslint-disable-line
```

Comment:
What's the `eslint-disable-line` for?  Can we do `eslint-disable-line some-eslint-rule` instead of disabling everything for this line?

Reply:
nevermind I guess haha

Reply:
`process` is only a global in node and this is browser code so we need to ignore it.  I couldn't figure out how to just allow a global for one line.